README.md

```diff
@@ -20,6 +20,22 @@ To get started with Kommunicate Android SDK, head over to the Kommunicate websit
 
 This is a sample that implements the Kommunicate android chat SDK. To use this sample you need to provide your application ID in app level build.gradle file's defaultConfig [Refer here](https://github.com/Kommunicate-io/Kommunicate-Android-Chat-SDK/blob/f78948edd81124847d1b6ee2179eadd968ec57b1/app/build.gradle#L13). Replace <Your-APP_ID> with the application ID obtained from Kommunicate dashboard.
 
+## Build a BOT on Kommunicate and integrate it in your Android app
+
+### Kompose
+
+[Kompose](https://dashboard.kommunicate.io/bots/bot-builder) is a Kommunicate’s bot builder that help you in building your own bot, a techie, non-techie, or a person who doesn’t have any idea about chatbots can also build the bot. Anyone can create a chatbot with the Kompose without any assistance.
```

Comment:
looks like there is extra whitespace before " is a "

Reply:
There is no extra space I checked.